app/uk/gov/hmrc/tai/connectors/BaseConnector.scala

```diff
@@ -43,11 +44,13 @@ abstract class BaseConnector(auditor: Auditor, metrics: Metrics, httpClient: Htt
     npsVersion
   }
 
-  def extraNpsHeaders(hc: HeaderCarrier, version: Int, txId: String): HeaderCarrier =
-    hc.withExtraHeaders("ETag" -> version.toString, "X-TXID" -> txId, "Gov-Uk-Originator-Id" -> originatorId)
-
   def basicNpsHeaders(hc: HeaderCarrier): HeaderCarrier =
-    hc.withExtraHeaders("Gov-Uk-Originator-Id" -> originatorId)
+    hc.withExtraHeaders(
```

Comment:
needs to be a `Seq[String, String]` of headers that we pass through explicitly to `httpClient.GET` in the getFromNPS call

Reply:
Done